docs/spec/protocol.rst

```diff
@@ -271,7 +272,8 @@ non-protocol generic types::
           ...
 
 ``Protocol[T, S, ...]`` is allowed as a shorthand for
-``Protocol, Generic[T, S, ...]``.
+``Protocol, Generic[T, S, ...]``. It is an error to include both
+``Protocol[T, S, ...]`` and ``Generic[T, S, ...]`` in a class definition.
```

Comment:
This should also be an error: `class C[T](Protocol[T]):`. The PEP 695 syntax creates an implicit inheritance from Generic, but I think it's better to be explicit.

Reply:
Added